index.html

```diff
@@ -1,1667 +1,2064 @@
-									<img class="background-image" alt="Lightbox Image" src="img/OTSDay_Vietnam.jpg">
-								</div>
-							</a>
-						</li>
-
-						<li>
-							<a href="img/DSC_0446_0.JPG" data-lightbox="true" data-title="Lightbox Image">
-								<div class="background-image-holder">
-									<img class="background-image" alt="Lightbox Image" src="img/fossasia-exhibition.jpg">
-								</div>
-							</a>
-						</li>
-
-						<li>
-							<a href="img/IMG_1146_0.JPG" data-lightbox="true" data-title="Lightbox Image">
-								<div class="background-image-holder">
-									<img class="background-image" alt="Lightbox Image" src="img/IMG_1146_0.JPG">
-								</div>
-							</a>
-						</li>
-
-						<li>
-							<a href="img/FOSSASIA 2011 Open Source Software Conference_0.JPG" data-lightbox="true" data-title="Lightbox Image">
-								<div class="background-image-holder">
-									<img class="background-image" alt="Lightbox Image" src="img/fossasia-memories.jpg">
-								</div>
-							</a>
-						</li>
-
-						<li>
-							<a href="img/FOSSASIA Social Event on Boat_0.JPG" data-lightbox="true" data-title="Lightbox Image">
-								<div class="background-image-holder">
-									<img class="background-image" alt="Lightbox Image" src="img/fossasia-bbq.jpg">
-								</div>
-							</a>
-						</li>
-
-						<li>
-							<a href="img/Girls in Open Source and IT FOSSASIA_0.JPG" data-lightbox="true" data-title="Lightbox Image">
-								<div class="background-image-holder">
-									<img class="background-image" alt="Lightbox Image" src="img/fossasia-indeed.jpg">
-								</div>
-							</a>
-						</li>
-
-						<li>
-							<a href="img/Free Open Source Summit Asia FOSSASIA_0.JPG" data-lightbox="true" data-title="Lightbox Image">
-								<div class="background-image-holder">
-									<img class="background-image" alt="Lightbox Image" src="img/fossasia-hackathon.jpg">
-								</div>
-							</a>
-						</li>
-					</ul>
-				</div>
-
-				<div class="container">
-					<div class="row">
-						<div class="col-sm-12">
-							<div>
-								<i class="icon pe-7s-camera text-white"></i>
-								<h1 class="text-white">Memories from FOSSASIA Events<br></h1>
-								<span class="lead text-white">Follow our photostream on <strong><a class="text-white" href="https://www.flickr.com/photos/fossasia/" target="_self">Flickr</a></strong></span>
-							</div>
-						</div>
-					</div>
-				</div>
-
-			</section>
-
-			<a id="mission" class="in-page-link"></a>
-
-			<section class="topics duplicatable-content">
-				<div class="container">
-
-					<div class="row">
-						<div class="col-sm-12">
-							<h1>Our Background and Mission<br></h1>
-						</div>
-					</div>
-
-					<div class="row">
-
-						<div class="col-sm-6 col-md-4">
-							<div class="topic">
-								<i class="pe-7s-loop icon-large"></i>
-								<h3>What We Do<br></h3>
-								<p class="lead">We bring amazing people together to create, develop, design and make things with Open T
```

Comment:
Hey @kushthedude and @mariobehling i changed the HTML for this div of row I didn't use any tool i just removed the button back to top issue's code sorry for the trouble